### ISSUE TITLE
how to train a teacher net?

### ISSUE THREAD

**Issue by Chrisable** [MIRALab-USTC/GraphAKD#5]
How to train a teacher net rather than downing the knowledge

**Comment by hhr114** [comment]
There was no secret. Just run teacher models on different datasets and save teacher logits as well as hidden representations. 
Here are some useful repos: [repo 1](https://github.com/chennnM/GCNII), [repo 2](https://github.com/zwt233/GAMLP), [repo 3](https://github.com/rusty1s/pyg_autoscale).
Hope it helps.

**Comment by Chrisable** [comment]
thank you!
